third_party/firebase/Firebase.framework/Headers/FAuthData.h

```diff
@@ -0,0 +1,50 @@
+//
```

Comment:
Rather than checking this binary blob into git, can we grab it from cloud storage?